hack/bicep-types-radius/src/autorest.bicep/test/integration/generated/basic/test.rp1/2021-10-31/types.json

```diff
@@ -59,7 +59,7 @@
         "type": {
           "$ref": "#/0"
         },
-        "flags": 1,
```

Comment:
Do we have any tests where we can cover this updated behavior?

Reply:
The integration tests check that the location field doesn't get flagged as required. We don't currently run those tests in workflows but we could update that if we want: https://github.com/radius-project/radius/blob/update-location-field/hack/bicep-types-radius/src/autorest.bicep/test/integration/integration.test.ts